crates/blockifier/src/execution/sierra_utils.rs

```diff
@@ -106,7 +106,9 @@ pub fn get_sierra_entry_function_id<'a>(
 
 pub fn setup_syscall_handler(
```

Comment:
Clippy throws the warning about too many arguments, maybe we can pass data as a structure?